help/delete-a-message.md

```diff
@@ -83,6 +83,14 @@ time, they are permanently and irrecoverably deleted from the Zulip
 server.  Server administrators can adjust the archival time using
 the `ARCHIVED_DATA_VACUUMING_DELAY_DAYS` setting.
 
+## Message notifications
+
+If you delete a message soon after sending it, any [pending email
+notifications](https://zulip.com/help/email-notifications#delay-before-sending-emails)
+for that message will be canceled, including @-mentions. However, deleting a
+message does not remove desktop alerts or mobile push notifications that have
+already been sent.
```

Comment:
A few notes here:
- The link can be relative.
- Should we also link to the articles on desktop and mobile notifications?
- Finally, I wonder about the use of "@-mentions". A quick git-grep on main shows we have three instances of this in the help center documentation (see code block below). I wonder if we just want to use "mentions" and link to the relevant article?

```console
$ git grep '@-mention' help/
help/dm-mention-alert-notifications.md:trigger email/push notifications as though they were personal @-mentions.  You
help/keyboard-shortcuts.md:* **Reply @-mentioning sender**: <kbd>@</kbd>
help/view-your-mentions.md:Topics with unread @-mentions are marked with an **@** indicator next to the number
```